<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GraphDigitiser</class>
 <widget class="QMainWindow" name="GraphDigitiser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>593</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Graph Digitiser</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGraphicsView" name="graphicsView"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QCheckBox" name="checkBox">
        <property name="text">
         <string>Automatic Mode</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="clearButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>593</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionAdd_Calibration"/>
    <addaction name="separator"/>
    <addaction name="actionSet_Axes_Scales"/>
   </widget>
   <widget class="QMenu" name="menuData">
    <property name="title">
     <string>Data</string>
    </property>
    <addaction name="actionView_Data"/>
    <addaction name="actionExport_Data"/>
   </widget>
   <addaction name="menuEdit"/>
   <addaction name="menuData"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSet_Axes_Scales"/>
  </widget>
  <action name="actionAdd_Calibration">
   <property name="text">
    <string>Add Calibration</string>
   </property>
  </action>
  <action name="actionSet_Axes_Scales">
   <property name="icon">
    <iconset resource="tip.qrc">
     <normaloff>:/img/img/icon.png</normaloff>:/img/img/icon.png</iconset>
   </property>
   <property name="text">
    <string>Set Axes Scales</string>
   </property>
  </action>
  <action name="actionView_Data">
   <property name="text">
    <string>View Data</string>
   </property>
  </action>
  <action name="actionExport_Data">
   <property name="text">
    <string>Export Data</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="tip.qrc"/>
 </resources>
 <connections/>
</ui>
